workspaces/src/network/sandbox.rs

```diff
@@ -135,36 +140,174 @@ impl Sandbox {
         ImportContractTransaction::new(id.to_owned(), worker.client(), self.client())
     }
 
-    pub(crate) async fn patch_state(
-        &self,
-        contract_id: &AccountId,
-        key: &[u8],
-        value: &[u8],
-    ) -> anyhow::Result<()> {
-        let state = StateRecord::Data {
-            account_id: contract_id.to_owned(),
-            data_key: key.to_vec(),
-            value: value.to_vec(),
+    pub(crate) fn patch_state(&self, account_id: AccountId) -> SandboxPatchStateBuilder {
+        SandboxPatchStateBuilder::new(self, account_id)
+    }
+
+    pub(crate) fn patch_account(&self, account_id: AccountId) -> SandboxPatchStateAccountBuilder {
+        SandboxPatchStateAccountBuilder::new(self, account_id)
+    }
+
+    // shall we expose convenience patch methods here for consistent API?
+
+    pub(crate) async fn fast_forward(&self, delta_height: u64) -> anyhow::Result<()> {
+        // NOTE: RpcSandboxFastForwardResponse is an empty struct with no fields, so don't do anything with it:
+        self.client()
+            // TODO: replace this with the `query` variant when RpcSandboxFastForwardRequest impls Debug
+            .query_nolog(&RpcSandboxFastForwardRequest { delta_height })
+            .await
+            .map_err(|err| anyhow::anyhow!("Failed to fast forward: {:?}", err))?;
+
+        Ok(())
+    }
+}
+
+//todo: review naming
+#[must_use = "don't forget to .apply() this `SandboxPatchStateBuilder`"]
+pub struct SandboxPatchStateBuilder<'s> {
```

Comment:
I think the word `Sandbox` is redundant here. Probably best to put patch related builders in [patch.rs](https://github.com/near/workspaces-rs/blob/main/workspaces/src/rpc/patch.rs) as mentioned in [122](https://github.com/near/workspaces-rs/pull/122#discussion_r853591005)